### PR TITLE
refactor: CLEN-555

### PR DESCRIPTION
**Pull request by MikeDobrzan**
refactor: Gradual deprecation of UUID

UUID has been marked as deprecated, and it's being renamed to UserId. For now, both variants will work, but UUID will be gradually phased out in favor of UserId.

### CODE REVIEW

File: PubNubUnity/Assets/PubNub/Workers/SubscriptionWorker.cs
Comment (budgetpreneur) [inline PubNubUnity/Assets/PubNub/Workers/SubscriptionWorker.cs:338]: UUID in "Authorized UUID doesn't match."?

File: PubNubUnity/Assets/PubNub/PubNubUnity/PNConfiguration.cs
Comment (budgetpreneur) [inline PubNubUnity/Assets/PubNub/PubNubUnity/PNConfiguration.cs:43]: Two invokes? I guess second one redundant. Just checking if all is good with the two invokes. Please confirm.

Comment (MikeDobrzan) [reply PubNubUnity/Assets/PubNub/PubNubUnity/PNConfiguration.cs:43]: Both events are still present - once we fully commit to removing UUID, this invoke will be deleted as well. I guess I could just subscribe one event to the other, but that would be mostly equivalent.

File: PubNubUnity/Assets/PubNub/Workers/SubscriptionWorker.cs
Comment (MikeDobrzan) [reply PubNubUnity/Assets/PubNub/Workers/SubscriptionWorker.cs]: Fixed the message.

File: PubNubUnity/Assets/PubNub/PubNubUnity/PNConfiguration.cs
Comment (MikeDobrzan) [reply PubNubUnity/Assets/PubNub/PubNubUnity/PNConfiguration.cs]: On second thought - I made it more consistent, so the double invocation has been removed.

**Review by budgetpreneur — APPROVED** [review]
Approving the PR. Please review  first/top comment which reflects in the docs change log when released. Also if anything merged to master before this PR, that changelog may need update here in the first/top comment. Please check with me or other code-owner before merge, for sanity check.

**Comment by MikeDobrzan** [comment]
@client-engineering-bot release

**Comment by parfeon** [comment]
@client-engineering-bot release

**Review by mohitpubnub — APPROVED** [review]

**Review by techwritermat — APPROVED** [review]

**Pull request merged**
